patternlibrary.html

```diff
@@ -367,5 +360,23 @@ <h2>Lorem ipsum!</h2>
   </section>
 </main>
 <script src="patternlibrary.js"></script>
+<script>
+    let carousel_image_wrapper = document.getElementById('carousel-image-wrapper')
+    function slide(image){
+      carousel_image_wrapper.style.transform = `translateX(-${image.id}00%)`
+    }
+      setTimeout(function (){
+        // alert('hello')
+        carousel_image_wrapper.style.transform = `translateX(-100%)`
+  
+      },2000)
+  
+      setTimeout(function (){
+        // alert('hello')
+        carousel_image_wrapper.style.transform = `translateX(-200%)`
+  
+      },12000)
+    
+    </script>
```

Comment:
Instead of putting the script here, you can put in the pattern library component section this:



Then in the Pattern library to get it to show up, you can put this:

<includes>
 <script src="/components/organism/image-carousel.js></script>
</includes>

Reply:
yes. it makes it better. so, we don't copy past the same file again and again. i will fix that now. Thank you